src/components/Home/Banner.tsx

```diff
@@ -0,0 +1,20 @@
+import { useNavigate } from "react-router-dom";
+
+const Banner = () => {
+  const navigate = useNavigate();
+  return (
+    <section className="banner">
+      <div className="banner-text">
+        <h1>ExoBotanica: plants from another world</h1>
+        <p>
+          You will not find these plants in any other shop on Earth. They have
+          been brought from a distant planet and are now available for you to
+          purchase.
+        </p>
+        <button onClick={() => navigate("/products")}>Shop Now</button>
```

Comment:
Try to use anchor (<a>) tags instead of buttons for navigation in the HTML, this is more accessible and more semantically correct